commec/tools/blast_tools.py

```diff
@@ -331,16 +328,16 @@ def trim_edges(df):
     return df, rerun
 
 
-def tophits(blast2):
+def get_top_hits(blast: pd.DataFrame):
```

Comment:
I always wondered why there were so many references to the same blast dataframe (with blast3 = blast2 etc etc) I figured they were mutable, and would just change the same space in memory. Good to see them gone.

Reply:
My guess is that this was just a refactor of an earlier, more pipeline-y script (where progressing through different processing steps as blast1 -> blast2 -> blast2 improved readability) but I'm not sure. I don't think there should be a memory advantage to doing it that way, but I'm not sure